e3nn/point/message_passing.py

```diff
@@ -29,3 +31,209 @@ def message(self, x_j, k):
         if k.shape[0] == 0:  # https://github.com/pytorch/pytorch/issues/37628
             return torch.zeros(0, k.shape[1])
         return torch.einsum('eij,ej->ei', k, x_j)
+
+
+class MinimalNetwork(torch.nn.Module):
```

Comment:
This seems more fitting as a test rather than module in the repository as most likely folks will not want to use a MinimalNetwork.

Reply:
True. This is a temporary class for development purposes that I used to debug calls to cuda extensions, whether gradients are computed and such.